spec/controllers/users/totp_setup_controller_spec.rb

```diff
@@ -32,10 +30,6 @@
           subject.current_user.decorate.qrcode(subject.user_session[:new_totp_secret])
         ).not_to be_nil
       end
-
-      it 'presents a QR code to the user' do
```

Comment:
Added a view spec at  57ac88fd

This is not testing the actual QR code generation. Created a separate GH issue for us to do that since it feels out of scope for this particular PR